pages/signin.js

```diff
@@ -154,7 +155,8 @@ class SigninPage extends React.Component {
           routes={this.getRoutes()}
           enforceTwoFactorAuthForLoggedInUser={enforceTwoFactorAuthForLoggedInUser}
           submitTwoFactorAuthenticatorCode={(values, actions) => {
-            this.props.login(this.props.token, values.twoFactorAuthenticatorCode);
+            const localStorage2FAToken = getFromLocalStorage(LOCAL_STORAGE_KEYS.ACCESS_TOKEN);
```

Comment:
Let's try to be consistent in the way we're naming things. Seems "2FA" is avoided in all other variable names.